codecov.yml

```diff
@@ -19,6 +32,19 @@ coverage:
         threshold: 0
         paths:
           - homeassistant/components/*/config_flow.py
+      required:
+        target: 100
+        threshold: 0
+        paths:
+          - homeassistant/components/*/device_action.py
+          - homeassistant/components/*/device_condition.py
```

Comment:
Why don't we group these paths with the config flows setting that has the same settings?

Reply:
Not a bad idea 👍

Reply:
Done